### PR TITLE
gog.com | website presence

### PR DESCRIPTION
**Pull request by Dom-in**
image
image
image

Profile Settings
On:
image
Off:
image

Money Settings:
On:
image
Off:
image

### CODE REVIEW

File: websites/G/gog/presence.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+const browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo"
+  };
+
+  const showmoney = await presence.getSetting("money");
+  const showprofile = await presence.getSetting("profiles");
```
Comment (ririxidev) [inline websites/G/gog/presence.ts:13]: merge consts

File: websites/G/gog/presence.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
+const presence = new Presence({
+  clientId: "788755903838027776"
+});
+
+const browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (ririxidev) [inline websites/G/gog/presence.ts:5]: merge consts

File: websites/G/gog/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  },
+  "service": "gog",
+  "description": {
+    "en": "Download the best classic and new games on Windows, Mac & Linux. A vast selection of titles, DRM-free, with free goodies and 30-day money-back guarantee."
+  },
+  "url": [
+    "gog.com",
+    "www.gog.com"
+  ],
+  "version": "1.0.1",
```
Comment (ririxidev) [inline websites/G/gog/dist/metadata.json:14]: 1.0.0

**Review by ririxidev — CHANGES_REQUESTED** [review]
and show proof of working settings in presence too :)

File: websites/G/gog/presence.ts [hunk 4]
```diff
@@ -0,0 +1,66 @@
+const presence = new Presence({
+  clientId: "788755903838027776"
+});
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo"
+  };
+
+  const showmoney = await presence.getSetting("money"),
```
Comment (ririxidev) [inline websites/G/gog/presence.ts:10]: still not merged

File: websites/G/gog/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    "en": "Download the best classic and new games on Windows, Mac & Linux. A vast selection of titles, DRM-free, with free goodies and 30-day money-back guarantee."
+  },
+  "url": [
+    "gog.com",
+    "www.gog.com"
+  ],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/zBN90ek.png",
+  "thumbnail": "https://i.imgur.com/HdKNcBK.png",
+  "color": "#FFFFFF",
```
Comment (ririxidev) [inline websites/G/gog/dist/metadata.json:17]: give it better color

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/G/gog/dist/metadata.json [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    "en": "Download the best classic and new games on Windows, Mac & Linux. A vast selection of titles, DRM-free, with free goodies and 30-day money-back guarantee."
+  },
+  "url": [
+    "gog.com",
+    "www.gog.com"
+  ],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/zBN90ek.png",
+  "thumbnail": "https://i.imgur.com/HdKNcBK.png",
+  "color": "#0000ff",
```
Comment (ririxidev) [inline websites/G/gog/dist/metadata.json:17]: once again, why blue? use this #4c1785

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/G/gog/presence.ts [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+    presenceData.details = "Looking At Game:";
+    if(!document.querySelector("body > div.layout.ng-scope > div:nth-child(8) > div.product-actions.hide-when-content-is-expanded.ng-scope > div.product-actions-price.product-actions-price--discounted > span.product-actions-price__discount.ng-binding")) {
+    presenceData.state = document.querySelector("body > div.layout.ng-scope > div:nth-child(8) > div.productcard-basics.hide-when-content-is-expanded > h1").innerHTML;
+    } else {
+    presenceData.state = document.querySelector("body > div.layout.ng-scope > div:nth-child(8) > div.productcard-basics.hide-when-content-is-expanded > h1").innerHTML +" ("+ document.querySelector("body > div.layout.ng-scope > div:nth-child(8) > div.product-actions.hide-when-content-is-expanded.ng-scope > div.product-actions-price.product-actions-price--discounted > span.product-actions-price__discount.ng-binding").innerHTML +")";
+   } else if (document.location.pathname.startsWith("/news/")) {
+    presenceData.details = "Reading:";
+    presenceData.state = document.querySelector("body > div.universe > div.wrapper.cf > div > article > h1 > b").textContent;
```
Comment (Bas950) [inline websites/G/gog/presence.ts:24]: looks like you got some syntax error in your ifs here

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/G/gog/presence.ts [hunk 8]
```diff
@@ -1,57 +1,58 @@
 const presence = new Presence({
-  clientId: "788755903838027776"
+  clientId 788755903838027776
```
Comment (Bas950) [inline websites/G/gog/presence.ts]: no?

File: websites/G/gog/presence.ts [hunk 9]
```diff
@@ -1,57 +1,58 @@
...
 });
 
-presence.on("UpdateData", async () => {
-  const presenceData: PresenceData = {
-    largeImageKey: "logo"
```
Comment (Bas950) [inline websites/G/gog/presence.ts]: no?

**Review by Bas950 — CHANGES_REQUESTED** [review]
idk what editor you use but you f*ed it all xD

**Pull request closed without merging**